src/App.tsx

```diff
@@ -69,28 +77,60 @@ const App: React.FunctionComponent = () => {
     };
   }, []);
 
+  return <div className="Root"></div>;
+};
+
+const router = createBrowserRouter([
+  {
+    path: '/*',
+    Component: Layout,
+    children: [
```

Comment:
Should this even have a path? The migration guide FAQ suggests it should be pathless.